Floofbot/Modules/Administration.cs

```diff
@@ -306,23 +314,26 @@ public async Task RemoveAutoBan([Summary("user ID")] string userId)
                 Moderator = $"{Context.User.Username}#{Context.User.Discriminator}",
                 ModeratorId = Context.User.Id,
                 Reason = reason,
-                UserId = badUser.Id,
+                UserId = (badUser != null) ? badUser.Id : uid,
```

Comment:
Optional refactor: Cool trick when searching up the question marks in C# code, you can use the [null coalescing operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator) to condense this to `UserId = badUser?.Id ?? uid` (though I always find it makes the code look very confused haha)

Reply:
Reading up on the rest of the changes, it seems like it might be beneficial to pull this logic out and give it its own variable as to prevent logic duplication. You could probably add an `else` condition to R307 to assign the `badUser.Id` value to `uid`

This should apply for the duplicates at R336, R387, and R394 as well.